Cargo.toml

```diff
@@ -25,12 +24,12 @@ secp256k1 = { version = "0.12", features = ["rand"] }
 serde = "1"
 serde_json = "1"
 shiplift = { version = "0.6", default-features = false }
-signal-hook = { version = "0.1", features = ["tokio-support"] }
 structopt = "0.3"
-tar = "0.4.26"
+tar = "0.4"
 tempfile = "3.1.0"
 thiserror = "1"
-tokio = "0.1"
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
I am guessing it'd save some compile time if you only select the necessary features.